M2/Macaulay2/packages/Macaulay2Doc/functions/random-doc.m2

```diff
@@ -77,12 +77,14 @@ Node
   Description
     Text
       If the @TT "Height"@ option specifies a number @TT "h"@ and @TT "T"@
-      is @TO "ZZ"@ then the integers returned are in the range @TT "[0, h)"@;
-      for @TO "QQ"@ the numerator and denominator are in the range @TT "[1, h]"@.
+      is @TO "ZZ"@ then the integers returned are in the range @TT "[0, h)"@.
+      If @TT "T"@ is @TO QQ@, then the results are drawn from the standard
+      uniform distribution and rounded to the nearest rational number with
+      denominator bounded by @TT "h"@.
     Example
       random RR
       random CC_100
-      tally for i to 100 list random GF 11
```

Comment:
I hope GF is memoized, no?

Reply:
No, it's not

Reply:
I would prefer leaving this line unchanged and, separately, memoize GF.